portal/models/questionnaire.py

```diff
@@ -0,0 +1,47 @@
+"""Questionnaire module"""
+from sqlalchemy.dialects.postgresql import JSONB
+
+from ..database import db
+
+
+class Questionnaire(db.Model):
+    __tablename__ = 'questionnaires'
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.Text, nullable=False, unique=True)
```

Comment:
Also, would prefer if this attribute had some name to indicate that it's a unique identifier or code. I feel like title implies human-readable